Misc/NEWS.d/next/Library/2019-07-31-16-49-01.bpo-37723.zq6tw8.rst

```diff
@@ -0,0 +1 @@
+Fix performance regression on regular expression parsing
```

Comment:
Maybe add more details? Like "... on parsing regular expression with huge character set".

Please add also "Patch by *your name*." at the end and add your name in `Misc/ACKS`.

Good first contribution!

Reply:
> Please add also "Patch by your name." at the end and add your name in Misc/ACKS.

Thanks for the reminder!